{"author":{"id":"06e2468b4cbb2d9ac1e0bb88f197d613005a7a4443cdcb9d576b8743df57b1fc"},"ops":[{"type":6,"timestamp":1562424375,"metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdDI0NTc1Mjk0NA=="},"nonce":"pWfeevoHGrpEInmgIm6wnFZxbDU=","target":"453dc001be94198566a5380eb8e7e6ca73c48cfb80932826e4d5fa6cd6f557b2","message":"When exporting new events to Github, creating and retrieving bugs using the graphql API can lead to some unexpected behaviour (only in some special cases):\n- creating a label `bug` then trying to create a label `Bug` will return an error `Name has already been taken`\n- having only one label named `bug` and trying to retrieve a label with name `Bug` will return a null ID.\n\nFor now to avoid this problem, try to not use two labels with the same letters like `core` and `CorE`.\n\nI wrote a message to the Github support, i'll keep this issue up to date once i have a response.","files":null},{"type":5,"timestamp":1562456594,"metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDI0NjQxNDM0MDI="},"nonce":"eCCX5lMXmzEE6HHSj5R/GL/yhrI=","added":["bridge"],"removed":[]},{"type":5,"timestamp":1562456594,"metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDI0NjQxNDM0MDM="},"nonce":"z2YUgFKwrm3AekqKSKkIN7axalg=","added":["bug"],"removed":[]},{"type":4,"timestamp":1566169350,"metadata":{"github-id":"MDExOkNsb3NlZEV2ZW50MjU2NDcyMTE3Mg=="},"nonce":"vH/Z7On/3z3Vj+7OJd/Rt0T25Ng=","status":2}]}